modules/models.py

```diff
@@ -55,7 +56,12 @@ def find_model_type(model_name):
     elif any((k in model_name for k in ['gpt4chan', 'gpt-4chan'])):
         return 'gpt4chan'
     else:
-        return 'HF_generic'
+        config = AutoConfig.from_pretrained(f"{shared.args.model_dir}/{model_name}")
```

Comment:
It appears that model_dir or model_name already has a `/`. This is breaking the install for me. Consider os.path.join for this type of concatenation.

Reply:
I missed that, should be fixed now https://github.com/oobabooga/text-generation-webui/commit/9c2e7c0fab385c8208a17a19feaa43cacf44bcf1